packages/backend/src/prompt/decision-loop-prompts.ts

```diff
@@ -43,6 +43,17 @@ A JSON object containing:
 - **Maintain continuity**: Acknowledge what's currently displayed
 - **Make informed choices**: Use state to determine next appropriate actions
 
+### Interactable Components
+
+Tambo allows developers to place components on screen for you to help the user interact with. 
```

Comment:
I thin you need to call out "Interactable Components" as an explicit term, otherwise it just feels like it's an adjective, like you are talking about components that are interactable vs being read-only.


> Tambo's "Interactable Components", or sometimes just "Interactables" are...

And then in the code you need to be consistent about capitalization just so that it is totally unambiguous what you're referring to.